src/components/InvertedFlatList/index.android.js

```diff
@@ -23,6 +13,8 @@ export default forwardRef((props, ref) => (
         style={styles.invert}
         ListFooterComponentStyle={styles.invert}
         verticalScrollbarPosition="left" // We are mirroring the X and Y axis, so we need to swap the scrollbar position
-        CellRendererComponent={InvertedCell}
+        CellRendererComponent={CellRendererComponent}
+        // Enabling absolute positioning with zIndex in the list. Source: https://reactnative.dev/docs/0.71/optimizing-flatlist-configuration#removeclippedsubviews
```

Comment:
The blank line before the comment is not a blocker, but it'd be good to update this comment to also explain _why_ we're doing this
```suggestion

        // Enabling absolute positioning with zIndex in the list. Source: https://reactnative.dev/docs/0.71/optimizing-flatlist-configuration#removeclippedsubviews
```

Reply:
Added the `why` aspect and restyled comment according to https://github.com/airbnb/javascript#comments--multiline